File: template/v8/example/glfw.js
import {GLFW} from 'raia_glfw';

const glfw = new GLFW();

glfw.init();
glfw.windowHint(GLFW.CLIENT_API, GLFW.OPENGL_ES_API);
glfw.windowHint(GLFW.CONTEXT_VERSION_MAJOR, 3);
glfw.windowHint(GLFW.CONTEXT_VERSION_MINOR, 0);
glfw.windowHint(GLFW.OPENGL_PROFILE, GLFW.OPENGL_CORE_PROFILE);
glfw.windowHint(GLFW.CONTEXT_CREATION_API, GLFW.EGL_CONTEXT_API);
glfw.setErrorCallbackAlt();
glfw.setJoystickCallbackAlt();

const window = glfw.createWindow(800, 600, "title");

while(glfw.windowShouldClose(window) === 0) {
    glfw.swapBuffers(window);
    glfw.pollEvents();
}

File: template/luajit/example/glfw.lua
local glfw = require("modules/glfw")

glfw.init()
glfw.windowHint(glfw.CLIENT_API, glfw.OPENGL_ES_API)
glfw.windowHint(glfw.CONTEXT_VERSION_MAJOR, 3)
glfw.windowHint(glfw.CONTEXT_VERSION_MINOR, 0)
glfw.windowHint(glfw.OPENGL_PROFILE, glfw.OPENGL_CORE_PROFILE)
glfw.windowHint(glfw.CONTEXT_CREATION_API, glfw.EGL_CONTEXT_API)
glfw.setErrorCallbackAlt()
glfw.setJoystickCallbackAlt();

local window = glfw.createWindow(800, 600, "title")

while glfw.windowShouldClose(window) == 0 do
    glfw.swapBuffers(window)
    glfw.pollEvents()
end